app/controllers/groups_controller.rb

```diff
@@ -337,9 +337,7 @@ def add_members
     end
 
     if emails.any?
-      if SiteSetting.enable_discourse_connect?
-        raise Discourse::InvalidParameters.new(I18n.t("groups.errors.no_invites_with_discourse_connect"))
-      elsif !SiteSetting.enable_local_logins?
+      if !SiteSetting.enable_local_logins? && !SiteSetting.enable_discourse_connect?
```

Comment:
We can remove this check here because the forum is already in a bad state if local logins are disabled without any altenative login methods

Reply:
Actually I checked, this is still valid because you can have local logins and discourse connect disabled but still have external auths configured

Reply:
For that scenario we dont' need to raise an error though. Invites can still be allowed.

Reply:
I had a look and it seems like this PR added it https://github.com/discourse/discourse/pull/11950 because of concerns raised here https://meta.discourse.org/t/bypass-sso-by-adding-unknown-email-to-group/177339. Now that we are allowing invites via SSO, we can still allow this when local logins are disabled and discourse connect is enabled, but if they are both disabled we should error. The original check had nothing to do with 3rd party auths

Reply:
Oooo the error is for no invites without local logins not when discourse connect is enabled. My bad.